### PR TITLE
cdc/capture: init table actor system if it is enabled

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve? 
it's a part of the table pipeline actor refactor.
This PR init table actor system if it is enabled

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/capture/capture.go [hunk 1]
```diff
@@ -60,6 +61,9 @@ type Capture struct {
 	etcdClient *etcd.CDCEtcdClient
 	grpcPool   kv.GrpcPool
 
+	tableActorSystem *actor.System
+	tableActorRouter *actor.Router
```
Comment (overvenus) [inline cdc/capture/capture.go:65]: Could you wrap them in a struct? like https://github.com/pingcap/ticdc/pull/3406

**Review by overvenus — APPROVED** [review]
rest LGTM

File: cdc/capture/capture.go (hunk 1 above)
Comment (sdojjy) [reply cdc/capture/capture.go:65]: ok

File: cdc/capture/capture.go [hunk 2]
```diff
@@ -354,6 +369,10 @@ func (c *Capture) AsyncClose() {
 	if c.grpcPool != nil {
 		c.grpcPool.Close()
 	}
+	if c.tableActorSystem != nil {
+		err := c.tableActorSystem.Stop()
+		log.Warn("stop table actor system failed", zap.Error(err))
```
Comment (liuzix) [inline cdc/capture/capture.go:374]: Do you think it would be necessary to set `c.tableActorSystem` to `nil`?

Comment (liuzix) [reply cdc/capture/capture.go:374]: I think the function `AsyncClose` is assumed to be idempotent.

Comment (sdojjy) [reply cdc/capture/capture.go:374]: yes, it's better to set  c.tableActorSystem to nil.

**Review by liuzix — APPROVED** [review]

**Pull request merged**